app/templates/macros/page_macros.html

```diff
@@ -2,30 +2,37 @@
     <div id="editor-{{ editable_html_obj.editor_name }}" contenteditable="false">
         {{ editable_html_obj.value|safe }}
     </div>
-
     {% if current_user.is_admin() %}
-       <button class="ui button start-edit">
+        <button class="ui button start-edit">
         Edit
-       </button>
-       <button class="ui primary button end-edit">
+        </button>
+        <p class= "updated_alert">Updated successfully</p>
+        <button class="ui primary button end-edit">
         Save
-       </button>
+        </button>
+        <button class="ui primary button cancel-edit">
+        Cancel
+        </button>
     {% endif %}
-
     <script>
     var editorIDName = "editor-{{ editable_html_obj.editor_name }}";
+    var original_data;
     $(document).ready(function() {
         {% if current_user.is_admin() %}
             $(".end-edit").hide();
+            $(".cancel-edit").hide();
+            $(".updated_alert").hide();
         {% endif %}
         $(".start-edit").click(function() {
             CKEDITOR.disableAutoInline = true;
             var editor = CKEDITOR.inline(editorIDName, {
```

Comment:
Oh sorry I thought that i replied to just already. We had replace but the text box would not become bigger. So it was an issue with semantic UI. So I used inline instead so that I could make the text editor box bigger.